src/utils/generator/ReplicaFiller.cpp

```diff
@@ -175,7 +175,7 @@ void ReplicaFiller::readXML(XMLfileUnits& xmlconfig) {
 void ReplicaFiller::init() {
 	ParticleContainerToBasisWrapper basisContainer;
 	std::shared_ptr<Object> object = std::make_shared<ObjectShifter>(_object, _origin);
-	basisContainer.setBoundingBox(object);
+	basisContainer.setBoundingBox(nullptr);
```

Comment:
this line becomes very confusing. Again, leave the old code commented out, but in.